package.json

```diff
@@ -84,6 +84,7 @@
     "@fontsource/roboto": "^5.0.8",
     "@mui/icons-material": "^5.14.18",
     "@mui/material": "^5.14.18",
+    "@react-hook/window-size": "^3.1.1",
```

Comment:
i should have known a hook like this existed... i've needed this countless times!

Reply:
Yep! Based on your other suggestion which lead to https://github.com/jeremyckahn/farmhand-shuffle/commit/6fc2588ba88e02bbbcad06008fe320271d9e4c04, I've since switched to [`usehooks-ts`'s variant](https://usehooks-ts.com/react-hook/use-window-size) since we're using the [`useDebounceCallback`](https://usehooks-ts.com/react-hook/use-debounce-callback) hook that it provides as well.